Kitodo/src/main/java/org/kitodo/production/metadata/MetadataEditor.java

```diff
@@ -75,6 +84,46 @@ public static void addLink(IncludedStructuralElement parentIncludedStructuralEle
         children.add(position, includedStructuralElement);
     }
 
+    /**
+     * Connects two processes by means of a link. The link is sorted as a linked
+     * included structural element in a included structural element of the
+     * parent process. The order is based on the order number specified by the
+     * user. This method does not create a link between the two processes in the
+     * database, this must and can only happen when saving.
+     *
+     * @param process
+     *            the parent process in which the link is to be added
+     * @param insertionPosition
+     *            at which point the link is to be inserted
+     * @param childProcessId
+     *            Database ID of the child process to be linked
+     * @throws IOException
+     *             if the METS file cannot be read or written
+     */
+    public static void addLink(Process process, String insertionPosition, int childProcessId) throws IOException {
```

Comment:
This method gets other arguments, namely the process and the specification of the insertion position, but no order, because we want to move away from it. It does more than the other method, especially opening and saving the METS file. If the use of the order has been removed from the metadata editor (there is already a ticket for it) both methods can be combined, that is, this method would call the other one.